src/mixin.js

```diff
@@ -18,7 +18,10 @@ var _extend = require('./internal/_extend');
  *
  *      R.mixin({ 'name': 'fred', 'age': 10 }, { 'age': 40 });
  *      //=> { 'name': 'fred', 'age': 40 }
+ *
+ *      var resetToDefault = R.mixin(R.__, {x: 0, y: 0});
+ *      resetToDefault({x: 5, y: 2}); //=> {x: 0, y: 0}
```

Comment:
It would be nice for at least one of the map entries in the lower-precedence map to show through:

```javascript
var resetX = R.mixin(R.__, {x: 0});
resetX({x: 5, y: 2}); //=> {x: 0, y: 2}
```